src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 800;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+  font-weight: 300;
+}
+
+.search-bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-input {
```

Comment:
use the rule `font-family: inherit` for input classes, so you can pass the tests.
Inputs and their placeholders don't inherit font family by default.